### PR TITLE
Added support for Almalinux 8

### PR DESCRIPTION
**Pull request by AliSajid**
*Issue #, if available:*

*Description of changes:*

I have added explicit release names for AlmaLinux 8.4 and Rocky Linux 8.3 and 8.4. Earlier, the mount helper was not working as it did not recognize that these releases also do not have libwrap just like RHEL 8 and CentOS 8. I have verified that it works on AlmaLinux and Rocky Linux on the specified versions.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by lukosbo** [comment]
Thanks for the PR. We’ll have someone take a look.

**Comment by Cappuccinuo** [comment]
Hey @AliSajid Thanks for the PR! Can you address the conflict, and merge the commit into one?

File: README.md [hunk 1]
```diff
@@ -10,6 +10,8 @@ The `efs-utils` package has been verified against the following Linux distributi
 | ------------ | ------------ | ------------- |
 | Amazon Linux 2017.09 | `rpm` | `upstart` |
 | Amazon Linux 2 | `rpm` | `systemd` |
+| Almalinux 8.4 | `rpm` | `systemd` |
+| RockyLinux 8.4 | `rpm` | `systemd` |
```
Comment (Cappuccinuo) [inline README.md:14]: We will add the distribution in README once we finish certain validation on our side. For now we can perform the rest of the change, so please revert the change in README for now, thanks.

**Review by Cappuccinuo — CHANGES_REQUESTED** [review]
LGTM, please combine the commits into single commit, thanks.

**Comment by AliSajid** [comment]
Hi. I'll get that done right away.

File: src/mount_efs/__init__.py [hunk 2]
```diff
@@ -241,10 +241,14 @@
 FEDORA_RELEASE_NAME = 'Fedora release'
 OPEN_SUSE_LEAP_RELEASE_NAME = 'openSUSE Leap'
 SUSE_RELEASE_NAME = 'SUSE Linux Enterprise Server'
+ALMALINUX_8_4_RELEASE_NAME = 'AlmaLinux release 8.4 (Electric Cheetah)'
+ROCKYLINUX_8_3_RELEASE_NAME = 'Rocky Linux release 8.3'
+ROCKYLINUX_8_4_RELEASE_NAME = 'Rocky Linux release 8.4'
```
Comment (Cappuccinuo) [inline src/mount_efs/__init__.py]: Also I believe rocky linux 8 will be supported after we merge https://github.com/aws/efs-utils/pull/103, so we can just focus on Almalinux for this PR.

File: src/mount_efs/__init__.py [hunk 3]
```diff
@@ -241,10 +241,14 @@
 FEDORA_RELEASE_NAME = 'Fedora release'
 OPEN_SUSE_LEAP_RELEASE_NAME = 'openSUSE Leap'
 SUSE_RELEASE_NAME = 'SUSE Linux Enterprise Server'
+ALMALINUX_8_4_RELEASE_NAME = 'AlmaLinux release 8.4 (Electric Cheetah)'
```
Comment (Cappuccinuo) [inline src/mount_efs/__init__.py:244]: Is that possible we just do `AlmaLinux release 8` here? So that we can support AlmaLinux 8.x, not only 8.4

File: .circleci/config.yml [hunk 4]
```diff
@@ -178,6 +178,18 @@ workflows:
       - build-deb-package:
           name: debian11
           image: debian:bullseye
+      - build-rpm-package:
+          name: rockylinux8.4
+          image: rockylinux/rockylinux:8.4
+      - build-rpm-package:
+          name: almalinux8.4
+          image: almalinux:8.4
```
Comment (Cappuccinuo) [inline .circleci/config.yml:186]: I think we can drop these in next revision, for the minor version

File: src/mount_efs/__init__.py (hunk 3 above)
Comment (AliSajid) [reply src/mount_efs/__init__.py:244]: I tried doing that and back then it wasn't working. I can add a separate line for supporting 8.x.

File: src/mount_efs/__init__.py (hunk 2 above)
Comment (AliSajid) [reply src/mount_efs/__init__.py]: Done

File: README.md (hunk 1 above)
Comment (AliSajid) [reply README.md:14]: Reverted.

File: .circleci/config.yml (hunk 4 above)
Comment (AliSajid) [reply .circleci/config.yml]: Done

**Review by AliSajid — COMMENTED** [review]
Reviewed and fixed

**Review by Cappuccinuo — CHANGES_REQUESTED** [review]
Thanks. Can you follow https://github.com/aws/efs-utils/pull/103 for the similar change, and squash all the commits into one? Thanks

File: src/mount_efs/__init__.py [hunk 5]
```diff
@@ -280,24 +279,8 @@
     MACOS_BIG_SUR_RELEASE,
     ORACLE_RELEASE_NAME,
     ALMALINUX_8_RELEASE_NAME,
+    ROCKY8_RELEASE_NAME
 ]
-=======
```
Comment (Cappuccinuo) [inline src/mount_efs/__init__.py]: I don't think this is a clean rebase right?

File: src/mount_efs/__init__.py [hunk 6]
```diff
@@ -280,24 +279,8 @@
     MACOS_BIG_SUR_RELEASE,
     ORACLE_RELEASE_NAME,
     ALMALINUX_8_RELEASE_NAME,
+    ROCKY8_RELEASE_NAME
```
Comment (Cappuccinuo) [inline src/mount_efs/__init__.py:282]: nit: Missed a `,` at the end

File: src/mount_efs/__init__.py [hunk 7]
```diff
@@ -256,7 +256,31 @@
 WATCHDOG_SERVICE = "amazon-efs-mount-watchdog"
 
 # MacOS instances use plist files. This files needs to be loaded on launchctl (init system of MacOS)
-
+<<<<<<< HEAD
```
Comment (Cappuccinuo) [inline src/mount_efs/__init__.py]: Same here, not a clean commit

File: src/mount_efs/__init__.py (hunk 5 above)
Comment (AliSajid) [reply src/mount_efs/__init__.py]: It is not. I have completely messed up the history while squashing.

**Review by Cappuccinuo — CHANGES_REQUESTED** [review]
The commits are not clean, we should have just one commit, with no squash record in each.

Also the commit message is not informative, should be something like

`Add Alma Linux to skip no libwrap list and circleci config`

File: src/mount_efs/__init__.py (hunk 7 above)
Comment (AliSajid) [reply src/mount_efs/__init__.py]: Would it help if I reforked and sent another pull request?

Comment (Cappuccinuo) [reply src/mount_efs/__init__.py]: Yes, that will also help.

Comment (Cappuccinuo) [reply src/mount_efs/__init__.py]: Also if you don't mind, I can submit the PR instead, which should be pretty quick.
